docs/apm-data-security.asciidoc

```diff
@@ -455,4 +483,97 @@ Please see the relevant agent's documentation for more information and examples:
 * Node.js: {apm-node-ref-v}/agent-api.html#apm-add-filter[`addFilter()`].
 // * PHP: {apm-php-ref-v}[]
 * Python: {apm-py-ref-v}/sanitizing-data.html[custom processors].
-// * Ruby: {apm-ruby-ref-v}/[]
+* Ruby: {apm-ruby-ref-v}/api.html#api-agent-add-filter[`add_filter()`].
+
+// ****************************************************************
+
+[[data-security-delete]]
+==== Delete sensitive data
+
+If you accidentally ingest sensitive data, follow these steps to remove or redact the offending data:
+
+. Stop collecting the sensitive data.
+Use the *remedy* column of the <<sensitive-fields,sensitive fields>> table to determine how to stop collecting
+the offending data.
+
+. Delete or redact the ingested data. With data collection fixed, you can now delete or redact the offending data:
++
+* <<redact-field-data>>
+* <<delete-doc-data>>
+
+[float]
+[[redact-field-data]]
+===== Redact specific fields
+
+To redact sensitive data in a specific field, use the {ref}/docs-update-by-query.html[update by query API].
```

Comment:
Is this the best way? What about the two examples below? Are there better ones that you can think of?

Reply:
Examples LGTM

Reply:
Thanks 😃